### PR TITLE
external/webclient : Header data and body data separation

### PR DESCRIPTION
**Pull request by JeonginKim**
fix error in downloaded data
separate header and body data

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: external/webserver/http_client.c [hunk 1]
```diff
@@ -291,9 +291,16 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
...
 
 					response->total_len = len->content_len;
-					response->entity_len = buf_len;
 
-					entity = *body;
+					if (is_header == true) {
+						response->entity_len = buf_len - len->sentence_start;
+						response->entity = buf + len->sentence_start;
+					}
+					else {
```
Comment (sunghan-chang) [inline external/webserver/http_client.c]: coding rule violation

File: external/webserver/http_client.c [hunk 2]
```diff
@@ -331,20 +338,23 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
...
 						if (response->entity) {
+							response->entity = *body;
 							response->entity_len = len->content_len;
 						}
 					}
-
-					if (is_header == true && !is_chunked) {
-						response->entity_len = 0;
-						HTTP_LOGD("[HEADER_BODY]buf_len : %d len->sentence_start : %d len->message_len : %d len->content_len %d entity_len : %d\n", buf_len, len->sentence_start, len->message_len, len->content_len);
+					else if (is_header == true && !is_chunked) {
```
Comment (sunghan-chang) [inline external/webserver/http_client.c]: violation

File: external/webserver/http_client.c [hunk 3]
```diff
@@ -331,20 +338,23 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
...
+						response->entity_len = buf_len - len->sentence_start;
+						response->entity = buf + len->sentence_start;
+						HTTP_LOGD("[HEADER_BODY]buf_len : %d len->sentence_start : %d len->message_len : %d len->content_len %d entity_len : %d\n", buf_len, len->sentence_start, len->message_len, len->content_len, response->entity_len);
 					}
-
-					entity = *body;
+					else {
```
Comment (sunghan-chang) [inline external/webserver/http_client.c]: violation

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**